### PR TITLE
use authz login on session restore

### PR DESCRIPTION
**Pull request by nihr43**
fixes the session restore dialog.

I've tested 'restore' by hitting restore within five minutes and running a query;
I've tested letting the dialog itself timeout which sends you back to heron.

my editor seems to have added a newline on 201.  are we ok with that?

### CODE REVIEW

**Comment by dckc** [comment]
> my editor seems to have added a newline on 201. are we ok with that?

yes, though ideally whitespaces changes are made in a separate commit.

**Comment by dckc** [comment]
I don't see how this is a "kumc hack"; what about it is specific to our site? Is it not a bug fix that would be reasonable to integrate upstream?

File: js-i2b2/cells/PM/PM_misc.js [hunk 1]
```diff
@@ -74,7 +74,11 @@ i2b2.PM.model.IdleTimer.subscribe("idle", function(){
 						
 				i2b2.h.LoadingMask.show();
 
-				i2b2.PM.doLogin();
+				// kumc heron login hack
+				var creds = i2b2.PM.pre_authz();
+				i2b2.PM.authz_login(creds);
+
+				//i2b2.PM.doLogin();
```
Comment (dckc) [inline js-i2b2/cells/PM/PM_misc.js]: Any particular reason to leave the old code around in comments?

**Comment by nihr43** [comment]
well, upstream doesn't use pre_authz() or authz_login() from what iv'e seen.

**Comment by dckc** [comment]
yes, and upstream is buggy, right?

I would much prefer that we rebase on our [partners-git][] branch, i.e. [i2b2:master][] so that we can make an upstream PR. If they don't think this is a good approach, I'd like to know.

But I guess it's not critical.

[partners-git]: https://github.com/kumc-bmi/i2b2-webclient/tree/partners-git
[i2b2:master]: https://github.com/i2b2/i2b2-webclient

**Review by dckc — APPROVED** [review]
ah... so `pre_authz` is part of the KUMC extension for CAS.

**Review by willsvankumc — APPROVED** [review]
Per our discussion and old code removal 👍

**Pull request merged**